charts/kong/templates/service-kong-cluster.yaml

```diff
@@ -0,0 +1,51 @@
+{{- /*
+Note that "http.enabled" should never be true for this service--the feature
+does not support plaintext mode.
+
+As such, it does not exist in the default values.yaml and is omitted from the
+initial render/do not render logic at the top, which is different from other
+Service templates.
+*/ -}}
+{{- if .Values.deployment.kong.enabled }}
+{{- if and .Values.cluster.enabled .Values.cluster.tls.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kong.fullname" . }}-cluster
```

Comment:
`cluster` is a terrible name and it is unfortunate that Kong decided to go with it. It is far too generic to have any meaning. Can we rename this to be `control` or `control-plane` instead?

Reply:
IMO we cannot--I'm pretty strongly against tools on top of Kong using their own names for Kong concepts or overriding defaults, from the standpoint that inconsistency will be even more confusing. If it's a concern, it needs to be addressed with the core team, not here.

If we have that conversation with them, we should probably also bring up the odd default port, which follows the `800X` pattern for plaintext traffic even though it's TLS and would use `844X` following the convention set by the others.